src/backends/fixerio/index.js

```diff
@@ -30,8 +29,8 @@ class FixerIoBackend {
 
     this.rates = {}
     this.currencies = []
-    const ledgerPairs = config.get('tradingPairs')
-    this.currencyPairs = ledgerPairs.map((p) => [p[0].slice(0, 3),
+    this.pairs = opts.pairs
```

Comment:
Basically `pairs` is on the form `[["USD@https://host.com/", "EUR@https://host2.com"]]` and `currencyPairs` will be `[["USD", "EUR"]]` so it's different variables